src/amber/cli/templates/scaffold/controller/crecto/spec/controllers/{{name}}_controller_spec.cr.ecr

```diff
@@ -1,71 +1,86 @@
 <% field_name = @fields.first.name -%>
 require "./spec_helper"
 
-def create_subject
-  subject = <%= class_name %>.new
-  subject.<%= field_name %> = "test"
-  subject.save
-  subject
+def create_model
+  model = <%= class_name %>.new
+  <% if @fields.first.reference? -%>
+  model.<%= field_name %> = <%= field_name.camelcase %>.new 
+  <% else -%>
+  model.<%= field_name %> = "test"
+  <% end -%>
+  model.save
+  model
 end
 
-describe <%= class_name %>Controller do
+class <%= class_name %>ControllerTest < Spec::Controller::Test
+  getter handler : Amber::Pipe::Pipeline
+
+  def initialize
+    @handler = Amber::Server.settings.handler
+    @handler.prepare_pipelines
+  end
+end
```

Comment:
Is this working? That's great if it does. What about CSRF? Is that defaulted to be part of `Amber::Server.settings.handler`?